manifests/init.pp

```diff
@@ -386,10 +386,12 @@
             $libvirtd_source                = $one::params::libvirtd_source,
             $rubygems                       = $one::params::rubygems,
             ) inherits one::params {
+  include one::prerequisites
   include one::install
   include one::config
   include one::service
 
+  Class['one::prerequisites']->
   Class['one::install']->
   Class['one::config']->
```

Comment:
Should we go for containment and add start.- and end-anchors? See also unordered included classes below